packages/tools/pie-icons-webc/package.json

```diff
@@ -33,13 +33,14 @@
     "PIE"
   ],
   "scripts": {
-    "build": "cp ./PieIconComponent.ts ./icons && vite build && tsc",
-    "build:icons": "node ./build.js",
+    "build": "node ./build.js && cp ./PieIconComponent.ts ./icons && vite build && tsc",
```

Comment:
This was the primary cause of the test failure. If no `icons` `dist` package was present, we define in turbo that a build is required before test run:

```
    "test:browsers": {
      "cache": true,
      "dependsOn": [
        "build",
        "test:browsers-setup"
      ],
      "inputs": [
        "$TURBO_DEFAULT$",
        "../../../apps/pie-storybook/stories/testing/icons.test.stories.ts"
      ],
      "outputs": [
        "__snapshots__/**"
      ]
    },
```

However a race condition was present where the `cp ./PieIconComponent.ts ./icons` command hadn't finished in time.